### PR TITLE
Setup and document auto-packaging and the release process.

### PR DESCRIPTION
**Pull request by Brondahl**

### CODE REVIEW

**Comment by dhedey** [comment]
LGTM, pending any name changes from https://github.com/Softwire/LochNessBuilder/pull/7

File: README.md [hunk 1]
```diff
@@ -291,3 +291,35 @@ Lots of method/type names have changed between v2.0 and v3.0, but there's very l
...
+* Update the READMEs
+  * Update the `.\README.md` file in general, and if appropriate add a "Migration" section to it.
+  * Update the `.\nugetREADME.md` file, with an appropriate selection from them main `README.md` file.
+* Create the new package.
+  * Clean and Rebuild the solution, which will automatically package everything up for you.
+    * The newly created package will be dropped in the `<root>\nuget` folder.
+  * Delete the previous version which will be sitting beside the newly produced version.
+    * Note that a clean copy of the previous version will already have been kept, in `<root>\nuget\ArchivedPackages`, when it was released. Double check that it's there and if not, look through git to recreate it.
+* Deploy to Nuget!
+  * Log into nuget with an account that is part of the LochNess organisation (Fet an existing member of the org to add you if not already included)
```
Comment (dhedey) [inline README.md:314]: Worth a link to the [LochNess Organisation](https://www.nuget.org/organization/LochNess)?

Fet > Get. Also, worth mentioning the names and usernames of the current LochNess organisation admins? And as a fallback, the email address for the LochNess Org routes to Softwire Helpdesk.

**Review by dhedey — APPROVED** [review]
All good excepting minor typo :)

File: README.md (hunk 1 above)
Comment (Brondahl) [reply README.md:314]: Done.

**Comment by Brondahl** [comment]
All changes done; ready to close.

**Comment by Brondahl** [comment]
All work reviewed, markups applied, code merged and latest version released.

**Pull request closed without merging**